packages/server/test/PartnerDishes.test.ts

```diff
@@ -13,6 +13,10 @@ const RESTAURANT_ID = '6200218668fc82e7bdf15088';
 describe(`${PATH}`, () => {
   const fixture = initE2eFixture();
 
+  afterEach(async () => {
+    await fixture.db.dishModel.deleteMany();
+  });
+
```

Comment:
Z tego co się orientuję to nie jest potrzebne od #89, ale nie dam sobie ręki uciąć.

Reply:
Specyfika tego testu nie pozwala mi tego usunąć. Bo podczas testu wykorzystuje GET, a on nie pozwala mi sprawdzic konkretnego dania, tylko dostaje cała liste. Więc trzeba zostawić.

Reply:
Prawda 👍

Chociaż generalnie możesz zawsze napisać test tak, żeby testował założenia z przypadku użycia w prawdziwym działaniu aplikacji, tzn. nie opierając się na niektórych założeniach.

Niżej w komentarzu daję przykład, np. PUT nie gwarantuje, że edytujemy pierwszy przedmiot w GETcie, tylko dowolny (`expect.arrayContaining`) (to też nie jest idealne, bo mamy paginację, więc przy ponad `n` elementach, ten edytowany nie znajdzie się na stronie 1).

Podobnie założenie "po utworzeniu 3 dishy, lista dishy będzie zawierać dokładnie 3 dishe" można zmienić na "będzie zawierać *co najmniej 3 dishe*", albo, jeszcze lepiej, na "będzie zawierać w sobie konkretnie 3 obiekty o takiej i takiej strukturze".